### ISSUE TITLE
create a logo for the Event Board

### ISSUE THREAD

**Issue by reshamas** [data-umbrella/event-board-web#215]
The Event Board is an open source project (using Django, React, Javascript, postgres db):
https://github.com/data-umbrella/event-board-web

This is the website of the Event Board application:
https://events.dataumbrella.org/

We are looking for a logo. The organization who is running this project is Data Umbrella, and more information about our community is here:  https://www.dataumbrella.org/

We are looking for a logo to encompass the umbrella as well as something related to events.

### Pull Requests
PRs with draft logo designs should be submitted to this repo:
https://github.com/data-umbrella/info

### Reference
- https://opensourcedesign.net/jobs/jobs/2022-09-26-design-a-logo-for-an-open-source-project

**Comment by akhilsharmaa** [comment]
I would like to work on this.

**Comment by reshamas** [comment]
> I would like to work on this.

@akhilsharmaa I added some more background to the description.

**Comment by akhilsharmaa** [comment]
ok, I'm working on this...

**Comment by Speeddesign1** [comment]
Alright 
I'm taking the project forward

**Comment by Speeddesign1** [comment]
> > This the project still going on? I would love to make some contribution
> 
> @Speeddesign1 
> Sure, please feel free to work on this.  Let us know if you have any questions.  

> @reshamas 
> how many of us are working on this project? how can we get reviews?

**Comment by reshamas** [comment]
@Speeddesign1 
There are two people working on this.

Let's try this:
a) submit a form to join the Data Umbrella Discord:  https://www.dataumbrella.org/
b) designs can be shared in our hacktoberfest channel

**Comment by Speeddesign1** [comment]
> @Speeddesign1 
> There are two people working on this.
> 
> Let's try this:
> a) submit a form to join the Data Umbrella Discord:  https://www.dataumbrella.org/
> b) designs can be shared in our hacktoberfest channel
> 
> @reshamas 
> I have got the form filled and I would love to get updated on the current situation

**Comment by reshamas** [comment]
@Speeddesign1 sent email.

**Comment by Kanishk-K-U** [comment]
I would like to contribute, is this still open?

**Comment by reshamas** [comment]
> I would like to contribute, is this still open?

@Kanishk-K-U Yes, it is.  When you have a design, please email us at:  info@dataumbrella.org

**Comment by reshamas** [comment]
Hi @Kanishk-K-U @Speeddesign1 @akhilsharmaa,
If you were able to come up with any logo ideas for the Event Board, feel free to email us here:  info@dataumbrella.org
Thank you.

**Comment by sowmiya-venkatesan** [comment]
Hi @reshamas , I'd like to contribute to this issue, if it is still open, could you please assign me?
